internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -175,9 +177,13 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	// TODO: need to be refactored because we can error out earlier.
 	if len(aliasesFor) != 0 && !s.certImported {
 		return "", fmt.Errorf("cannot specify alias hosted zones when env certificates are managed by Copilot")
 	}
+	if len(aliasesFor) != 0 && s.cdnEnabled {
+		return "", fmt.Errorf("cannot specify alias hosted zones when cdn is enabled in environment %q", s.env)
```

Comment:
Should we add a `RecommendedAction` to this error? Telling the user to manage the A-record on their own to point to switch at the appropriate time between CDN and ALB.

Reply:
This `cdnEnabled` field seems to be here only for validation purpose (just like `certImported`). We either want to make it clear in the comment that both fields need to be refactored, or just move this `cdn` validation to maybe `cli/deploy` or `manifest` in this PR